src/Individual_XP.cpp

```diff
@@ -137,7 +145,7 @@ class Individual_XP_command : public CommandScript
         }
         else
         {
-            me->GetSession()->SendAreaTriggerMessage("Your current XP rate is %u", me->CustomData.GetDefault<PlayerXpRate>("Individual_XP")->XPRate);
+            me->GetSession()->SendAreaTriggerMessage("Your current XP multiplier is %u", me->CustomData.GetDefault<PlayerXpRate>("Individual_XP")->XPRate);
```

Comment:
why this change?

Reply:
Because it’s a multiplier rather than an actual rate , for instance if you set rate In worldserver.conf to 2 thane set personal do to 2 your rate is 4x at least players. It’s correct English translation so as not to confuse players . I learned this from my players and made the change accordingly

Reply:
ok interesting then why not